examples/example_savestate.m

```diff
@@ -0,0 +1,16 @@
+% Initialization.
```

Comment:
With this many examples, we might have to start categorizing them into subdirectories. Add separate issue for that?

Reply:
Good idea

Reply:
Added #56